tests/storage/storage_migration/conftest.py

```
@@ -276,83 +277,103 @@ def deleted_old_dvs_of_stopped_vms(unprivileged_client, namespace):
-            for cmd in [
-                f"sudo mkfs.ext4 {HOTPLUGGED_DEVICE}",
-                f"sudo mkdir {MOUNT_HOTPLUGGED_DEVICE_PATH}",
-                f"sudo mount {HOTPLUGGED_DEVICE} {MOUNT_HOTPLUGGED_DEVICE_PATH}",
-            ]
-        ],
-        wait_timeout=TIMEOUT_2MIN,
-        sleep=TIMEOUT_5SEC,
-    )
-    yield vm_for_storage_class_migration_with_hotplugged_volume
+def vm_with_mounted_hotplugged_disks(vm_for_storage_class_migration_with_hotplugged_volumes):
+    for device, mount_path in zip(HOTPLUGGED_DEVICES, MOUNT_HOTPLUGGED_DEVICE_PATHS):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)